modules/manage/pages/cluster-maintenance/cluster-balancing.adoc

```diff
@@ -3,46 +3,48 @@
-
-The following table summarizes the various balancing options in Redpanda:
-
-
-.Cluster balancing options
-[cols="4,2,11"]
+[.no-clip]
 |===
-|Balancer |xref:get-started:licenses.adoc[License] |Description
+| Balancer |Description
 
-|Partition leadership balancing
-|Community Edition, Enterprise Edition
-|Topic-aware balancer designed to distribute partition leadership across available nodes. This helps avoid topic leadership hotspots on one or a few specific nodes in your cluster.
+| <<Partition leadership balancing>>
+| Topic-aware balancing that transfers the leadership of a broker's partitions to other replicas. This helps avoid topic leadership hotspots on one or a few specific brokers in your cluster. 
+ +
+ +
+Redpanda automatically provides leadership balancing when brokers are added or decommissioned.
 
-|Partition replica balancing
-|Community Edition, Enterprise Edition
-|Topic-aware balancer designed to distribute partition replicas when adding nodes to the cluster. This helps avoid topic replica hotspots on one or a few specific nodes in your cluster.
+| <<Partition replica balancing>>
+| Topic-aware balancing that moves partition replicas. This helps avoid topic replica hotspots on one or a few specific brokers in your cluster. 
+ +
+ +
+When creating new topics or balancing existing partitions, Redpanda first distributes partition count evenly on each broker. Because different topics in a cluster can have vastly different load profiles, this balances the workload across brokers. Next, Redpanda balances the overall partition count across the cluster.
+ +
+ +
+Redpanda automatically provides partition replica balancing when brokers are added or decommissioned.
 
-|Continuous Data Balancing
-|Enterprise Edition
-|Balancer designed to monitor free disk space and proactively move data between nodes to avoid exhausting available disk space on any given node. This balancer does not keep the relative fullness of each node within a defined range, it just prevents hitting the fullness threshold of each individual node.
+| xref:manage:cluster-maintenance/continuous-data-balancing.adoc[Continuous Data Balancing]
+
+| With an Enterprise license, you can additionally enable _Continuous Data Balancing_ to continuously monitor broker and rack availability and disk usage. This enables self-healing clusters that dynamically balance partitions. It also ensures adherence to rack-aware replica placement policy and self-heals after rack (or availability zone) failure or replacement. This balancer does not keep the relative fullness of each broker within a defined range, it just prevents hitting the fullness threshold of each individual broker. 
```

Comment:
Why is Continuous Data Balancing italicized here?

Reply:
removed italics